docs/02-formalia/titlepage.rst

```diff
@@ -5,32 +5,181 @@
+
+*eProsima Fast DDS*:
+
+* Allows to pre-allocate resources, to minimize dynamic resource allocation.
+* Avoids the use of unbounded resources.
+* Minimizes the need to copy data.
+
+**Multi-platform.**
+
+The OS dependencies are treated as pluggable modules.
+The user can easily implement his platform modules to *eProsima Fast DDS* library in his specific platform.
+By default, the project can run over Linux and Windows.
+
+**Free and Open Source.**
+
+The Fast DDS library, the underneath RTPS library, the generator tool, the internal dependencies (such as *eProsima
+Fast CDR*) and the external ones (such as the *foonathan* library) are free and open source.
+
+**Easy to use.**
+
+The project comes with an out-of-the-box example, the *DDSHelloWorld* (TODO: add ref) that puts into communication a
+publisher and a subscriber, showcasing how *eProsima Fast DDS* is deployed.
+Additionally, the interactive demo *ShapesDemo* is available for the user to dive into the DDS world.
+Both the DDS and the RTPS layers are thoroughly explained in the Fast DDS section (TODO: add ref).
+
+**Best effort and reliable communication.**
+
+*eProsima Fast-DDS* supports both *best effort*, for fast and light communication, and *reliable*, when reliability
+is needed.
+
+**Built-in Discovery Service.**
+
+*eProsima Fast DDS* is based on the dynamical discover of the existence of publishers and subscribers
+and performs this task continuously without
+the need to contact any servers.
+
+**Plug-and-play Connectivity**
+
+New applications and services are automatically discovered, and can join and leave the network at any time without the
+need for reconfiguration.
+
+**Scalability and Flexibility**
+
+DDS builds on the concept of a global data space. The middleware is in charge of propagating the information between
+publishers and subscribers. This guarantees that the distributed network is adaptable to reconfigurations
+and scalable to a large number of entities.
+
+**Application Portability**
+
+The DDS specification includes a platform specific mapping to IDL, allowing an application using
+DDS to switch among DDS implementations with only a re-compile.
+
+**Extensibility**
+
+*eProsima Fast DDS* allows the protocol to be extended and enhanced with new services without
+breaking backwards compatibility and interoperability.
+
+**Configurability and Modularity**
+
+*eProsima Fast DDS* provides an intuitive way to be configured, either through code or XML profiles.
+Modularity allows simple devices to implement a subset of the protocol and still participate in
+the network.
+
+**Two API Layers**
+
+*eProsima Fast DDS* comprises a high-level publisher-subscriber layer focused on usability and a lower-level 
+writer-reader layer that provides finer access to the RTPS protocol.
+
+**Commercial support**
+
+Available at support@eprosima.com
+
+
+Structure of the Documentation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 This documentation is organized into the following sections:
```

Comment:
I think this section looks weird in the HTML, maybe the list in the end with a description would suffice

Reply:
Ok, so deleting lines 153-175, and maybe just leave the sentence:
'This documentation is organized into the following sections:'
?

Reply:
Yep, that's what I thought